src/views/blog-post/post-title-header/post-title-header.astro

```diff
@@ -30,7 +32,17 @@ const originalLinkStr = post.originalLink && new URL(post.originalLink).host;
 			<div class={style.date}>
 				<div class={style.date__published}>
 					<Icon width={24} height={24} name="date" aria-hidden="true" />
-					<p class="text-style-button-regular">{publishedStr}</p>
+					{
+						versions.length ? (
```

Comment:
```suggestion
						versions.length > 1 ? (
```
It seems like there's always be one version of each post, so the dropdown should only be shown if there's more than one.